paper.tex

```diff
@@ -45,12 +51,12 @@
 
 \usepackage{listings}
 
+\newcommand\email[1]{\href{mailto:#1}{#1}}
 \title{SymPy: Symbolic Computing in Python}
 
 \input{authors}
 
-\begin{document}
-\maketitle
+\keywords{Keyword1, Keyword2, Keyword3}
```

Comment:
The keywords I used for the preprint and SICOMP were "symbolic, python, computer algebra system"